cli/cmd/generate.go

```diff
@@ -384,6 +396,32 @@ func addWorkloadOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) e
 	return nil
 }
 
+func addSeedshareOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) error {
```

Comment:
```suggestion
func addSeedshareOwnerKeyToManifest(manifest *manifest.Manifest, keyPath string) error {
```
Or is the naming on purpose to not collide with the package name? This should not be a compiler error though, as it is scope-aware

Reply:
The package is also needed further down, so we can't shadow it.